shop/admin/delivery.py

```diff
@@ -182,8 +182,8 @@ def render_delivery_note(self, request, delivery_pk=None):
         return HttpResponse(content)
 
     def get_inline_instances(self, request, obj=None):
-        inline_instances = [OrderItemInlineDelivery(self.model, self.admin_site)
-                            if isinstance(instance, OrderItemInline) else instance
+        inline_instances = [
+            OrderItemInlineDelivery(self.model, self.admin_site) if isinstance(instance, OrderItemInline) else instance
             for instance in super(DeliveryOrderAdminMixin, self).get_inline_instances(request, obj)]
```

Comment:
I prefer to have the closing `]` on an extra line here.

Reply:
fixed